lenskit/batch/_multi.py

```diff
@@ -291,9 +291,12 @@ def _predict(self, rid, algo, test):
         return preds, watch.elapsed()
 
     def _recommend(self, rid, algo, test, candidates):
-        if self.recommend is None:
+        if self.recommend is None or self.recommend is False or self.recommend is 0:
```

Comment:
why not just `if not self.recommend`? That covers all the falsey values.

Reply:
I... did not realize that. How elegant.